packages/e2e-test-utils-playwright/src/editor/publish-post.ts

```diff
@@ -29,4 +29,9 @@ export async function publishPost( this: Editor ) {
 	await this.page.click(
 		'role=region[name="Editor publish"i] >> role=button[name="Publish"i]'
 	);
+
+	const urlString = await this.page.inputValue( 'text="Post address"' );
```

Comment:
Normally, we would prefer role selectors over text selectors if they're available.